content/ctl_ohai.md

```diff
@@ -24,6 +24,30 @@ aliases = ["/ctl_ohai.html"]
 
 The following examples show how to use the Ohai command-line tool:
 
+### Query for a specific attribute
+
+To query for an attribute with the ohai command-line tool, pass the attribute
+as an argument
+
+```bash
+ohai os
```

Comment:
It's probably worth mentioning what recipe style attributes these map to: `ohai os` being the same data as `node['os']`